tools/interactive/interactivetool_anylabeling.xml

```diff
@@ -0,0 +1,71 @@
+<tool id="interactive_tool_anylabeling" tool_type="interactive" name="AnyLabeling Interactive" version="@VERSION@" profile="24.2">
+    <description>Effortless data labeling with AI support from YOLO and Segment Anything! AnyLabeling = LabelImg + Labelme + Improved UI + Auto-labeling</description>
+    <requirements>
+	    <container type="docker">quay.io/sunyi000/docker-yololabel:v0.4.16</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="AnyLabeling" requires_domain="True">
+            <port>5800</port>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code">
+    <![CDATA[
+         export HOME=\$PWD &&
+         
+         mkdir -p ./home ./home/output ./home/input_images &&
+	 chown -R 1000:1000 ./home &&
+
+         #for $i,$img in enumerate($input_images):
+            ln -s "$img" "./home/input_images/${img.element_identifier}" &&
+         #end for
+
+         #if $classes:
+             mkdir -p ./home/input_classes &&
+             #for $i,$cls in enumerate($classes):
+                 ln -s "$cls" "./home/input_classes/${cls.element_identifier}" &&
+             #end for
+         #end if
+
+         #if $annotations:
+             mkdir -p ./home/input_annotations &&
+             #for $i,$ann in enumerate($annotations):
+                 ln -s "$ann" "./home/input_annotations/${ann.element_identifier}" &&
+             #end for
+         #end if
+
+         #if $models:
+             mkdir -p ./home/input_models &&
+             #for $i,$mo in enumerate($models):
+                 ln -s "$mo" "./home/input_models/${mo.element_identifier}" &&
+             #end for
+         #end if 
+
+        /init 
+         
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="input_images" type="data" format="jpg,png,tiff,bmp,gif,pcx,ppm,psd,pbm,pgm,eps" multiple="true"  label="Input images"/>
+        <param name="annotations" type="data" format="txt,json" optional="true" multiple="true" label="Pre-existing annotations"/>
+        <param name="classes" type="data" format="txt" optional="true" label="Classes file (one class per line)"/>
+        <param name="models" type="data" format="auto" optional="true" label="Auto-labeling model file"/>
```

Comment:
what are those models rechnically?

Reply:
it will be a mix of onnx model files and yml files, as described here https://anylabeling.nrl.ai/docs/custom-models

Reply:
ok, then we need to tar them up I think and provide here the tar ball

Reply:
i have changed this to tar format, so it takes 1 tar of model files and yaml files..